app/code/Magento/Developer/Model/Di/PluginList.php

```diff
@@ -143,6 +143,10 @@ private function addPluginToList($pluginInstance, $pluginMethod, $methodTypes, $
     {
         if ($methodTypes & $typeCode) {
             if (!array_key_exists($pluginInstance, $this->pluginList[$this->pluginTypeMapping[$typeCode]])) {
+                $this->pluginList[$this->pluginTypeMapping[$typeCode]][$pluginInstance] = [];
+            }
+
+            if (!in_array($pluginMethod, $this->pluginList[$this->pluginTypeMapping[$typeCode]][$pluginInstance])) {
```

Comment:
Hi @orlangur ,
What do you think about this modification? I implemented this second condition just because it looks like a cheapest and still technologically possible solution. Also, this method is totally isolated and used only for this command.

Reply:
Too wordy and not elegant as to me but still good enough :) Asymptotic  complexity became poor but should not be noticeable.

Good job with tiniest possible diff 👍